joomla/joomla-cms
# [4.3] CSP Supplementary to #39726

- Also intorduce the class `js-select-reset-on-change` that resets all the other filters except the target and submits the form

Pull Request for Issue #39726 .

### Summary of Changes
- Intorduces the class `js-select-reset-on-change` that resets all the other filters except the target and submits the form
- Fix the functionality for selects that are outside of the container `.js-stools-container-bar` and have either a class `js-select-reset-on-change` or `js-select-submit-on-change`

### Testing Instructions

Check the left most selects on the pages:
- administrator/index.php?option=com_menus&view=menus
- administrator/index.php?option=com_associations&view=associations
- administrator/index.php?option=com_languages&view=installed and all the other com_menus views
- administrator/index.php?option=com_modules&view=modules&client_id=0
- administrator/index.php?option=com_modules&view=modules&client_id=1
- administrator/index.php?option=com_templates&view=templates&client_id=0
- administrator/index.php?option=com_templates&view=templates&client_id=1
- administrator/index.php?option=com_templates&view=styles&client_id=0
- administrator/index.php?option=com_templates&view=styles&client_id=1

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request

Works

### Link to documentations
Please select:


@Quy
